docs/permissions/getting-started.md

```diff
@@ -119,43 +75,61 @@ Now that the permission backend is running, it’s time to enable the permission
      enabled: true
    ```
 
-2. Update the PermissionPolicy in `packages/backend/src/plugins/permission.ts` to disable a permission that’s easy for us to test. This policy rejects any attempt to delete a catalog entity:
+2. Its now all wired up and working, great! But perhaps we don't want to simply allow everything, lets try and create our own policy, to do this you can create a new folder in `packages/backend/src` called `permissions` and create a new file called `policy.ts`, in that file we can add the following to create a policy that denies deleting entities from the catalog:
```

Comment:
Is this the preferred way to add policies? (as we use OPA we don't do it :sweat_smile:)  I am not sure if you should `yarn new` and create a module for a policy to simply do it this way? LMK!

Reply:
We created a new backend module called `@internal/backstage-plugin-permission-backend-module-policy-[company-name]`. That follows closely to @backstage/plugin-**permission**-backend-**module**-allow-all-policy. We don't have enough policies to break them into separate plugins. 

Many of the doc examples show inlining the examples like you are doing, so I think that keeps things in sync. What about adding something at the end saying "while our example shows creating the policy in `src/backend`, it's best practice for permission policies to live in a backend module that extends the permissions plugin.

Reply:
Good one, can always show how to do it _both_ ways, I am not sure how maybe @freben or @Rugvip feel about the best practise here? Maybe you could weigh in when you have a moment? 😅

Reply:
I tend to just put it into the backend index as well ... there ends up being a couple of extra steps that add confusion otherwise. Matter of taste, but yeah, those instructions for adding finely granular packages start to add up once you document how to do it for a bunch of individual adaptations.

Reply:
But I have a separate thought. What if we cut out everything after the config enablement? We could basically instead say "congratulations, you have now ..." blah blah, and then "to actually start enabling fine grained permissions checks, see https://backstage.io/docs/permissions/writing-a-policy"

Reply:
> But I have a separate thought. What if we cut out everything after the config enablement? We could basically instead say "congratulations, you have now ..." blah blah, and then "to actually start enabling fine grained permissions checks, see https://backstage.io/docs/permissions/writing-a-policy"

Yeah I like this approach as well, let me play around with the wording see if we can make it flow nicely 😄